Autopilot/PathManager/Inc/pathStateClasses.hpp

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 #include "pathManagerStateManager.hpp"
+#include "cruisingState.hpp"
 #include "pathManager.hpp"
 #include "altimeter.hpp"
 #include "gps.hpp"
 #include "AutoSteer.hpp"
 #include "waypointManager.hpp"
+#include "pathDatatypes.hpp"
+#include "CommWithAttitudeManager.hpp"
+#include "trackFinder.hpp"
```

Comment:
Why is this guy here ? Shouldn't this just live inside Sensor fusion ? And Sensor fusion's output struct should just have a "Track" field

Reply:
yeah trackFinder shouldn't be there, should be only used in sensorFusion.

Reply:
This means Tony needs to make a PR to put trackFinder in SensorFusion.hpp

Reply:
kk, all you acc need to do is add the "track" field to Tony's struct, he'll make sure things work internally